.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thanks for submitting a pull request! Before pressing that tempting green submit
+button, please make sure you check off all the following boxes:
+
+- [ ] All existing unit tests pass in all supported Python versions (2.6, 2.7,
+      and 3.3+)
+- [ ] New tests have been added to verify the new functionality / bug fix
+- [ ] All code works on at least Windows, Mac OS X, and Linux
+- [ ] All code follows [PEP 8](https://www.python.org/dev/peps/pep-0008/), or
+      remains consistent with existing code.
+- [ ] You agree to release all submitted code under the
```

Comment:
We don't need this. We have not had this problem with contributors. Also what if this checkbox was unmarked the PR got merged? I feel like this checkbox more trouble than it is worth.